upload-chunked-file.js

```diff
@@ -0,0 +1,43 @@
+function chunkFileUpload(file, serverUrl, chunkSize) {
```

Comment:
Add boundary checks for `chunkSize` to ensure it is a sensible size for the application's use case. This will prevent unnecessary system resource usage.